recipes/controller.rb

```diff
@@ -16,14 +16,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+node.default['openstack']['endpoints']['compute-vnc-bind']['bind_interface'] =
+  Chef::DelayedEvaluator.new do
+    node['osl-openstack']['vnc_bind_interface']['controller']
+  end
```

Comment:
Ugh, I was worried about this. Not everything works with `Chef::DelayedEvaluator`, unfortunately. I guess just revert it for now.